framework/CMakeLists.txt

```diff
@@ -11,6 +11,7 @@ add_subdirectory(system)
 add_subdirectory(audio)
 add_subdirectory(midi)
 add_subdirectory(midi_old)
+add_subdirectory(rpc)
```

Comment:
I think no need separate module "rpc" or it should be a different name.
"rpc" - is just protocol of interaction. 
module "rpc" - is part of the audio module, and not used anywhere else.

Reply:
rpc is not an audio protocol and can be used for anything else.

Reply:
> "rpc" - is just protocol of interaction
I agree with Igor. 
Maybe better move it to thirdparty like the async module?

Reply:
In fact, you can't use it in another case.
If I send a message, only the audio worker will receive it.
To use in other cases, you need to create other instances.
To distinguish instances from each other, you need to have either different interfaces (for example, IAudioRpcServer) or have a factory of RPC servers.
The implementation of the RPC itself is very specific, I can't imagine what another case we can apply it.

Reply:
rpc module removed